tests/test_project/admin.py

```diff
@@ -73,12 +73,13 @@ class OperatorInline(HelpTextStackedInline):
 
 
 @admin.register(Project)
-class ProjectAdmin(UUIDAdmin, ReceiveUrlAdmin):
+class ProjectAdmin(CopyableFieldsAdmin, ReceiveUrlAdmin):
     inlines = [OperatorInline]
     list_display = ('name',)
     fields = ('uuid', 'name', 'key', 'receive_url')
-    readonly_fields = ('uuid', 'receive_url')
+    readonly_fields = ('receive_url',)
```

Comment:
I fixed the `add_page` exception in [`47b9f00` (#332)](https://github.com/openwisp/openwisp-utils/pull/332/commits/47b9f0079edb8bc6333a59fe90bb85fddad123c0), but I think `receive_url` won't work (shows empty) with `copyable_fields` because this field is explicitly defined by `ReceiveUrlAdmin`, which generates the `receive_url` field for the `ModelAdmin` object. see below:

https://github.com/openwisp/openwisp-utils/blob/9fd347fb861af674674a4bcc59c54bcdeef18b8c/openwisp_utils/admin.py#L124-L150

If there is something we can change to make it work, then please let us know.